src/LoginWithAuth0.ts

```diff
@@ -0,0 +1,89 @@
+import { Auth0Client } from "@auth0/auth0-spa-js"
+
+// Constant for the domain suffix
```

Comment:
Somewhat redundant comment 😄 

Also comments that apply to a specific variable/property should generally be written in the symbol documentation comment `/** */` style; this way vscode intellisense will pick up on them https://typedoc.org/guides/doccomments/